### PR TITLE
Fix run-jupyter-local to use the Leo docker image

### PR DESCRIPTION
**Pull request by calbach**
Use the Leo docker image instead of minimal-notebook, which is outdated. Make some adjustments to `run-jupyter-local.sh`; now fixed for me.

Have you read [CONTRIBUTING.md](../CONTRIBUTING.md) lately? If not, do that first.

I, the developer opening this PR, do solemnly pinky swear that:

- [x] I've documented my API changes in Swagger

In all cases:

- [ ] Get two thumbsworth of review and PO signoff if necessary
- [ ] Verify all tests go green
- [ ] Squash and merge; you can delete your branch after this
- [ ] Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

File: jupyter-docker/run-jupyter-local.sh [hunk 1]
```diff
@@ -1,6 +1,10 @@
 #!/bin/bash
+#
+# To rebuild the docker image, run:
+#   docker build jupyter-docker/ -t broadinstitute/leonardo-notebooks:local
+set -e
 
-DOCKER_IMG=jupyter/minimal-notebook
+DOCKER_IMG=broadinstitute/leonardo-notebooks:local
```
Comment (rtitle) [inline jupyter-docker/run-jupyter-local.sh]: 👍 , great to test with our docker image. Spark/Hail won't be available, but cool that it works anyway.

File: jupyter-docker/run-jupyter-local.sh [hunk 2]
```diff
@@ -12,13 +16,14 @@ start () {
     fi
 
     echo "Starting Jupyter server container..."
-    docker create -it --name ${CONTAINER} --rm -p 8888:8888 $DOCKER_IMG 'export PYTHONPATH=$PYTHONPATH:/etc/jupyter/custom;start-notebook.sh'
+    docker create -it --rm --name ${CONTAINER} -p 8000:8000 -e GOOGLE_PROJECT=project -e CLUSTER_NAME=cluster "${DOCKER_IMG}"
```
Comment (rtitle) [inline jupyter-docker/run-jupyter-local.sh]: This means the notebook server will listen at `http://localhost:8000/notebooks/project/cluster` :)

**Comment by calbach** [comment]
No problem. I don't have any permissions on this repo so I will leave this PR in your hands.

**Comment by rtitle** [comment]
I'm going to close this because the changes were included in https://github.com/DataBiosphere/leonardo/pull/259, which I merged.

**Pull request closed without merging**